style.css

```diff
@@ -882,32 +880,31 @@ textarea { resize: none; }
     display: flex;
     flex-direction: row;
     align-items: flex-start;
-    /* flex: 1fr 1fr 330px; */
+    flex: 1fr 1fr 330px;
     height: 500px;
     min-width: auto;
     padding: 0;
-    border-bottom: 2px solid #DFE1E6;
-    background-image: 
-    url(./Images/illustration-1-contact-form-desktop.svg),
-    url(./Images/illustration-2-contact-form-desktop.svg),
-    url(./Images/contact-form-illustratoin-3-desktop.svg);
+    border-bottom: 2px solid #dfe1e6;
+    background-image:
+      url(./Images/illustration-1-contact-form-desktop.svg),
+      url(./Images/illustration-2-contact-form-desktop.svg),
+      url(./Images/contact-form-illustratoin-3-desktop.svg);
     background-repeat: no-repeat;
-    background-position: 0px 230px, 350px 400px, right;
+    background-position: 0 230px, 350px 400px, right;
   }
 
   .contact-container {
     display: flex;
-    justify-content: space-around;
-    border: 2px solid red;
-    width: 70%;
+    flex-shrink: 1, 5;
+    justify-content: center;
+    margin-right: 5rem;
   }
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure to align correctly the contact section content correctly as on the provided Figma design. :wink: 

    - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/188013512-6c9b2ddb-b9fc-4e77-aa1a-17a6c5a14a2b.png)


    - ### Figma design
  
   ![image](https://user-images.githubusercontent.com/87197412/188013656-1e2ce1b4-275d-48ff-b67b-97741449d15c.png)

Reply:
Hi @levy002 Please help me understand what it means by the "correct" alignment. The previous code reviewer was specific in his request with regard to the contact form alignment, which has been corrected from column to row using flexbox. The task is to code the portfolio project "as close as possible to your chosen template" and so therefore it implies that it does not have to be 100% identical. A lot of time went into correcting such changes and I found it extremely difficult to replicate the desktop contact form (especially in terms of centering it) and tried my best to code it as close as possible all while fitting the different screen sizes. You are not providing me with any direction and are vague in your request. I kindly ask if you may give me specific instructions or consider a re-review.

Reply:
Hello @keebot-png, excellent for communicating well and politely. While I was testing your website I noticed that the `input` and `textarea` in your form are taking a long width than the `form` container itself. So to be more clear on this point kindly adjust the width of your `input` and even your `textarea` (message part). If you check it again you have given a long width to those form elements. I highly recommend you adjust it again in order to make it responsive. I have checked your website desktop version from `768px` to `1440px` breakpoints but this contact form container fits correctly the screen when the width is `1650px` which is out of the required range.